chart/kubeapps/templates/apprepository-jobs-cleanup-rbac.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.rbac.create -}}
+# The cleanup job is necessary to remove "apprepositories" created
+# in the bootstrap job or in the web application
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ template "kubeapps.apprepository-jobs-cleanup.fullname" . }}
+  annotations:
+    helm.sh/hook: pre-delete
```

Comment:
should this be post-delete? What if deletion fails?

Reply:
yes, it will fit better in the post-delete hook